design/scss/_variables.scss

```diff
@@ -24,14 +24,13 @@ $grays: (
 
 $navy: #333a64 !default;
 $navy-dark: #414254 !default;
-$cream: #f8e4c9 !default;
+$cream: #f8e4c9 !default; // do not use unless discussed first
 
 $primary: #333a64 !default;
 $secondary: #ff826b !default;
-$success: #00a19a !default;
-$warning: #ffbb59 !default;
-$danger: #ff5c6f !default;
-$info: #0dcaf0 !default;
```

Comment:
Is there a specific reason you removed this one?
It was a custom color I think and now applies the default bootstrap color which is darker than this one